### PR TITLE
WIP - add unit test for InstanceID.GetSlug

### PR DESCRIPTION
**Pull request by matthyx**
## PR Type:
Tests

___
## PR Description:
This PR introduces a new unit test for the GetSlug method of the InstanceID struct in the instanceidhandler package. The test checks the functionality of the method using a predefined set of fields and asserts the expected output.

___
## PR Main Files Walkthrough:
<details> <summary>files:</summary>

`instanceidhandler/v1/instanceidhandler_test.go`: Added a new unit test function TestInstanceID_GetSlug to test the GetSlug method of the InstanceID struct. The test uses a predefined set of fields and checks the output against the expected slug string.
</details>

### CODE REVIEW

File: instanceidhandler/v1/instanceidhandler_test.go [hunk 1]
```diff
@@ -202,3 +202,46 @@ func TestInstanceIDToDisplayName(t *testing.T) {
...
+				namespace:     tt.fields.namespace,
+				kind:          tt.fields.kind,
+				name:          tt.fields.name,
+				containerName: tt.fields.containerName,
+			}
+			got, err := id.GetSlug()
+			assert.Equalf(t, tt.want, got, "GetSlug()")
+			assert.ErrorIs(t, err, tt.wantErr)
+		})
+	}
```
Comment (vladklokun) [inline instanceidhandler/v1/instanceidhandler_test.go:246]: question: how is this different from the test exactly above?

Comment (matthyx) [reply instanceidhandler/v1/instanceidhandler_test.go:246]: right... didn't notice it when I clicked on generate unittest for fuction

Comment (matthyx) [reply instanceidhandler/v1/instanceidhandler_test.go:246]: lemme merge

Comment (vladklokun) [reply instanceidhandler/v1/instanceidhandler_test.go:246]: Before you do it, how this test case would be different from the existing ones?

To me it seems that it checks the exact same case where there is a valid input without anything special we should care about. So why add it at all?

Comment (matthyx) [reply instanceidhandler/v1/instanceidhandler_test.go]: to show Eran that we reproduce his particular case

Comment (matthyx) [reply instanceidhandler/v1/instanceidhandler_test.go]: plus we don't have any replicaset

Comment (vladklokun) [reply instanceidhandler/v1/instanceidhandler_test.go]: Nah, it’s okay. Thankfully, it’s a cheap and straightforward test.

**Review by vladklokun — APPROVED** [review]

**Pull request closed without merging**